thirdparty/quazip/CMakeLists.txt

```diff
@@ -33,9 +33,11 @@ src/zip.h
 add_library(quazip STATIC ${SRC_FILES})
 target_include_directories(quazip PUBLIC src)
 target_link_libraries(quazip PUBLIC zlib Qt5::Core)
+target_compile_definitions(quazip PUBLIC QUAZIP_STATIC)
```

Comment:
quazip doesn't to be static this library is used only on utilities.

Reply:
Heh, I was following the established pattern, most of the third party libs are compiled in static and shared mode :)

Reply:
Not all =) static only for a game binaries. Dynamic for editors.